schemas/gen-missing-crd-schemas.sh

```diff
@@ -2,7 +2,8 @@
 
 # run from root
 
-cd schemas/
```

Comment:
Please move it to separate PR

Reply:
Why? Not conflicting at all. Please don't create work for no reason

Reply:
but it is delaying feature delivery